### DISCUSSION TITLE
Project updates

### DISCUSSION THREAD

**Discussion by CodeGlitcher** [Farfetch/kafkaflow#274] [General] [answered]
Hi,

We are looking if we can intergratie kafka-flow in our application,
it looks great but we have some concerns about maintenance of this project.
It does not look its actively updated. (looking add comments on issue and pull request. Some dependencies updates are opend a few month ago.)
Are you still continuing to update this project or is it done?

Gr,
Rob

**Comment by filipeesch** [comment, marked as answer]
Hi @CodeGlitcher, we are planning the next releases of KafkaFlow and we are currently reorganizing the internal team responsible for the project. This project is alive, it is largely used at Farfetch and it will not die.
